cime_config/cesm/machines/config_lt_archive.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 
-<config_lt_archive version="1.0.0">
+<config_lt_archive version="1.0">
```

Comment:
version handling in cime is implemented as real and so 1.0.0 doesn't work here.  We can revisit this issue if there is strong objection and alternative to the current implementation.